tailcfg/tailcfg.go

```diff
@@ -2040,6 +2040,7 @@ const (
 	CapabilityDataPlaneAuditLogs NodeCapability = "https://tailscale.com/cap/data-plane-audit-logs" // feature enabled
 	CapabilityDebug              NodeCapability = "https://tailscale.com/cap/debug"                 // exposes debug endpoints over the PeerAPI
 	CapabilityHTTPS              NodeCapability = "https"                                           // https cert provisioning enabled on tailnet
+	CapabilityWebClient          NodeCapability = "webclient"                                       // allows starting web client in tailscaled
```

Comment:
I'm not sure which way this is meant to be: a temporary flag to turn it on for early pre-GA testers, or an emergency override flag to turn it off if there are problems?

If the former, maybe "preview-webclient"? If the latter, can we use the absence of this for on, and add an explicit one for if/when we ever disable it?

Reply:
ah yes... it's the former.  Renamed to "preview-webclient".